### PR TITLE
added basic VerifiablePresentation and Must functions

### PR DESCRIPTION
**Pull request by woutslakhorst**

### CODE REVIEW

**Review by dmeijboom — APPROVED** [review]
Nice :D

File: did/did.go [hunk 1]
```diff
@@ -110,6 +109,27 @@ func ParseDID(input string) (*DID, error) {
 	return did, nil
 }
 
+// Must accepts a function like Parse and returns the value without error or panics otherwise.
+func Must(fn func(string) (*DID, error), input string) DID {
```
Comment (stevenvegt) [inline did/did.go:113]: Should this be exported?

**Review by stevenvegt — APPROVED** [review]
Looks good! I wonder if the `Must` should be exported.

File: did/did.go (hunk 1 above)
Comment (woutslakhorst) [reply did/did.go:113]: It's very specific (given the signature) and all parse functions have already been covered. I'll make it private

**Pull request merged**